go.mod

```diff
@@ -7,51 +7,37 @@ require (
 	github.com/Azure/go-autorest/autorest/adal v0.9.18
 	github.com/Azure/go-autorest/tracing v0.6.0
 	github.com/dlclark/regexp2 v1.4.0
-	github.com/docker/cli v20.10.14+incompatible
-	github.com/docker/docker v20.10.14+incompatible
+	github.com/docker/cli v20.10.16+incompatible
 	github.com/golang-jwt/jwt/v4 v4.4.1
 	github.com/moby/term v0.0.0-20210619224110-3f7ff695adc6
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/cobra v1.4.0
 	github.com/stretchr/testify v1.7.1
+	oras.land/oras-go/v2 v2.0.0-20220516090534-31f7572a6976
```

Comment:
You may want to update the oras-go to the latest version, and re-run `go mod tidy` and `go mod vendor`.

Reply:
updated.